src/api/Models/Layer.cs

```diff
@@ -0,0 +1,488 @@
+
+    /// <summary>
+    /// Gets or sets the id of the <see cref="Layer"/>'s lithostratigraphy.
+    /// </summary>
+    [Column("lithostratigraphy_id_cli")]
+    public int? LithostratigraphyId { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Layer"/>'s lithostratigraphy.
+    /// </summary>
+    public Codelist? Lithostratigraphy { get; set; }
+
+    /// <summary>
+    /// Gets or sets the id of the <see cref="Layer"/>'s humidity.
+    /// </summary>
+    [Column("humidity_id_cli")]
+    public int? HumidityId { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Layer"/>'s humidity.
+    /// </summary>
+    public Codelist? Humidity { get; set; }
+
+    /// <summary>
+    /// Gets or sets whether the <see cref="Layer"/> is striae.
+    /// </summary>
+    [Column("striae_lay")]
+    public bool? IsStriae { get; set; }
+
+    /// <summary>
+    /// Gets or sets the id of the <see cref="Layer"/>'s unconrocks.
+    /// </summary>
+    [Column("unconrocks_id_cli")]
+    public int? UnconrocksId { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Layer"/>'s unconrocks.
+    /// </summary>
+    public Codelist? Unconrocks { get; set; }
+
+    /// <summary>
+    /// Gets or sets the id of the <see cref="Layer"/>'s lithok.
+    /// </summary>
+    [Column("lithok_id_cli")]
+    public int? LithokId { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Layer"/>'s lithok.
+    /// </summary>
+    public Codelist? Lithok { get; set; }
+
+    /// <summary>
+    /// Gets or sets the id of the <see cref="Layer"/>'s import.
+    /// </summary>
+    [Column("import_id")]
+    public int? Import { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Layer"/>'s instrument.
+    /// </summary>
+    [Column("instr_id")]
+    public string? Instrument { get; set; }
+
+    /// <summary>
+    /// Gets or sets the id of the <see cref="Layer"/>'s instrument kind.
+    /// </summary>
+    [Column("instr_kind_id_cli")]
+    public int? InstrumentKindId { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Layer"/>'s instrument kind.
+    /// </summary>
+    public Codelist? InstrumentKind { get; set; }
+
+    /// <summary>
+    /// Gets or sets the id of the <see cref="Layer"/>'s instrument status.
+    /// </summary>
+    [Column("instr_status_id_cli")]
+    public int? InstrumentStatusId { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Layer"/>'s instrument status.
+    /// </summary>
+    public Codelist? InstrumentStatus { get; set; }
+
+    /// <summary>
+    /// Gets or sets the id of the <see cref="Layer"/>'s instrument stratigraphy.
+    /// </summary>
+    [Column("instr_id_sty_fk")]
+    public int? InstrumentStratigraphyId { get; set; }
+
+    /// <summary>
+    /// Gets or sets the id of the <see cref="Layer"/>'s casing kind.
+    /// </summary>
+    [Column("casng_kind_id_cli")]
+    public int? CasngKindId { get; set; }
```

Comment:
REQ: Ich glaube das sollte `Casing` lauten, deshalb Typo fixen:
```suggestion
    public int? CasingKindId { get; set; }
```

Dito für die anderen `Casng`-Schreibweisen.